src/defaults.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 Simo Sorce
+// See LICENSE.txt file for terms
+
+//! This module contains constants and functions to populate information
+//! fields returned by PKCS#11 APIs
+
+use std::env;
+
+use crate::interface;
+
+/// Default slot description.
+pub const SLOT_DESCRIPTION: &str = "Kryoptic Slot";
+/// Default manufacturer ID string.
+pub const MANUFACTURER_ID: &str = "Kryoptic Project";
+/// Default token label (differs slightly if FIPS feature is enabled).
+#[cfg(feature = "fips")]
+pub const TOKEN_LABEL: &str = "Kryoptic FIPS Token";
+/// Default token label.
+#[cfg(not(feature = "fips"))]
+pub const TOKEN_LABEL: &str = "Kryoptic Soft Token";
+/// Default token model string (differs slightly if FIPS feature is enabled).
+#[cfg(feature = "fips")]
+pub const TOKEN_MODEL: &str = "FIPS-140-3 v1";
+/// Default token model string.
+#[cfg(not(feature = "fips"))]
+pub const TOKEN_MODEL: &str = "v1";
+
+pub fn hardware_version() -> interface::CK_VERSION {
+    interface::CK_VERSION { major: 0, minor: 0 }
+}
+
+pub fn firmware_version() -> interface::CK_VERSION {
```

Comment:
Please, add doc strings here

Reply:
doh!
I'll add an entry to the PR template that explicitly asks about doc string